manifests/a/APRELTehnologijad/o/o/ExetomsiConverterPro/3.9.0.3/APRELTehnologijad.o.o.ExetomsiConverterPro.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: APRELTehnologijad.o.o.ExetomsiConverterPro

```

Comment:
```suggestion
PackageIdentifier: APRELTehnologija.ExeToMsiConverterPro
```

Reply:
**Please make sure to update all of the folder and file names using GitHub Desktop on Windows from https://desktop.github.com**